src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./App.scss";
+import { Route } from "react-router";
+import SideBar from "./components/sidebar/sidebar.components";
+import Chatroom from "./components/chatroom/chatroom.components";
+import Profile from "./components/profile/profile.components";
+import Other from "./components/other/other.components";
+
+function App() {
+  return (
+    <div className="App">
+      <SideBar></SideBar>
```

Comment:
```suggestion
      <SideBar />
```

이 부분도 `self-closing tag`를 써 보면 어떨까요?